Disable TF_CORD_SUPPORT for CUDA due to CUDA 10 compiler issues

Author: Alexander Grund (TU Dresden)

diff --git a/tensorflow/core/platform/default/cord.h b/tensorflow/core/platform/default/cord.h
index 0dbdc051f4865..cdfab1e5da905 100644
--- a/tensorflow/core/platform/default/cord.h
+++ b/tensorflow/core/platform/default/cord.h
@@ -16,7 +16,12 @@ limitations under the License.
 #ifndef TENSORFLOW_CORE_PLATFORM_DEFAULT_CORD_H_
 #define TENSORFLOW_CORE_PLATFORM_DEFAULT_CORD_H_
 
+#if !defined(__CUDACC__)
+
+// TODO(frankchn): Resolve compilation errors when building absl::Cord with CUDA
 #include "absl/strings/cord.h"
 #define TF_CORD_SUPPORT 1
 
+#endif  // __CUDACC__
+
 #endif  // TENSORFLOW_CORE_PLATFORM_DEFAULT_CORD_H_
